### PR TITLE
OpenAPI: Get mempool transations - Removed default value and updated max and example values

### PR DESCRIPTION
**Pull request by LakshmiLavanyaKasturi**
Updated maximum and example values for the Get mempool transitions.
Removed default value.

The openAPI.yaml file is updated.

### CODE REVIEW

File: docs/openapi.yaml [hunk 1]
```diff
@@ -268,9 +268,8 @@ paths:
           required: false
           schema:
             type: integer
-            default: 96
-            maximum: 200
-            example: 100
+            maximum: 50
```
Comment (obycode) [inline docs/openapi.yaml:271]: Looks like the default is 20. Can we add that field back in?

Comment (LakshmiLavanyaKasturi) [reply docs/openapi.yaml:271]: Trying to learn,  "limit": 20, is the limit considered as the default value?

Comment (LakshmiLavanyaKasturi) [reply docs/openapi.yaml:272]: Done. Updated.

**Comment by LakshmiLavanyaKasturi** [comment]
Relates to https://github.com/hirosystems/docs/issues/299

**Review by rafaelcr — APPROVED** [review]
Thanks @LakshmiLavanyaKasturi

File: docs/openapi.yaml (hunk 1 above)
Comment (obycode) [reply docs/openapi.yaml:272]: `limit` is the name of the parameter. The default value is what this limit is set to if no parameter is passed.

**Pull request merged**